docs/tutorials/getting-started/python.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Python SDK and Flask
+description: Getting Started with the OpenFeature Python SDK on Flask
+---
+
+import FlagdContent from '@site/src/components/custom/tutorial/flagd-content.mdx';
+import FlagdChangeContent from '@site/src/components/custom/tutorial/flagd-change-content.mdx';
+import WhyDefaultContent from '@site/src/components/custom/tutorial/why-default-content.mdx';
+
+
+# Getting Started with the OpenFeature Python SDK
+
+## Introduction
+
+This walk-through teaches you the basics of using OpenFeature with Python.
+You'll learn how to:
+
+- Install the Python SDK
+- Install and configure a provider
+- Perform basic feature flagging
+
+## Requirements
+
+This walk-through assumes that:
+
+- You have a basic understanding of Python and Flask.
+- You have Flask 3.x and Pyhton 3.10 or later.
+- You have Docker installed and running on the host system.
+  The latest version of Docker can be found [here][docker-download].
+
+> **NOTE:** If you don't have docker installed, check the available options to install Flagd [here](https://flagd.dev/installation/).
+
+## Walk-through
+
+### Step 1: Create a a minimal flask application
+
+To get started, create a new folder, bootstrap the project, and install the dependencies.
+This can be done by running the following commands.
+
+```sh
+  mkdir openfeature-python-intro
+  cd openfeature-python-intro
+  pip install Flask
+```
+
+### Step 2: Create a Flask app
+
+Create a new file named `app.py` inside openfeature-python-intro directory and include the following code.
+
+```python
+  from flask import Flask
+  app = Flask(__name__)
+
+  @app.route("/")
+  def index():
+      return "Flask Server"
+```
+
+### Step 3: Add the OpenFeature SDK
+
+Let's install the OpenFeature SDK using the following commands.
+
+```sh
+  pip install typing-extensions
+  pip install openfeature-provider-flagd
+```
+
+Update `app.py` to import the SDK.
+```python
+  from openfeature import api
+  from openfeature.contrib.provider.flagd import FlagdProvider
+```
+
+Once you've imported `OpenFeature`, a new client can be created using the `FlagdProvider`.
+
+```python
+  api.set_provider(FlagdProvider())
+  client = api.get_client()
+```
+
+The client can now be used to get a feature flag value.
+In this case, we'll get a `boolean` value using the `welcome-message` [flag key][flag-key]
+and fallback value, which is returned if there's abnormal behavior using the `client.get_boolean_value()` method.
+
+```python
+@app.route("/")
+def index():
+    showWelcomeMessage = client.get_boolean_value("welcome-message", False)
```

Comment:
Please use `snake_case` for Python local variable names: `show_welcome_message`

Reply:
The flag definition is shared across all tutorials and uses `welcome-message`. We could update the flag key but it would impact all the other tutorials.